src/teleop/gui/src/components/PDBFuse.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+    <div class="wrap">
+      <div>
+        <h3>Power Distribution Board</h3>
+      </div>
+      <div class="table">
+        <table class="tableFormat" style="undefined;table-layout: fixed">
+          <colgroup>
+            <col style="width: 80px">
+            <col style="width: 100px">
+            <col style="width: 100px">
+          </colgroup>
+          <thead>
+            <tr>
+              <td class = "tableElement"></td>
+              <td class = "tableElement">Temperature</td>
+              <td class = "tableElement">Current</td>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td class = "tableElement">3.3V</td>
+              <td class = "tableElement" v-bind:style="{color: pdb_data[0].temp.color}">{{pdb_data[0].temp.val.toFixed(5)}}°C</td>
+              <td class = "tableElement" v-bind:style="{color: pdb_data[0].current.color}">{{pdb_data[0].current.val.toFixed(5)}}A</td>
+            </tr>
+            <tr>
+              <td class = "tableElement">5V</td>
+              <td class = "tableElement" v-bind:style="{color: pdb_data[1].temp.color}">{{pdb_data[1].temp.val.toFixed(5)}}°C</td>
+              <td class = "tableElement" v-bind:style="{color: pdb_data[1].current.color}">{{pdb_data[1].current.val.toFixed(5)}}A</td>
+            </tr>
+            <tr>
+              <td class = "tableElement">12V</td>
+              <td class = "tableElement" v-bind:style="{color: pdb_data[2].temp.color}">{{pdb_data[2].temp.val.toFixed(5)}}°C</td>
+              <td class = "tableElement" v-bind:style="{color: pdb_data[2].current.color}">{{pdb_data[2].current.val.toFixed(5)}}A</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+    </template>
+    
+    <script>
+    import ROSLIB from "roslib"
+    const pdb_temp_limit = 100
+    const pdb_current_limits = [//TBD
+      2,
```

Comment:
Is this still TBD or did EHW say that it was a good estimate?

Reply:
This is still TBD at the moment.